lib/src/models/account.dart

```diff
@@ -1,65 +1,84 @@
 import 'emails.dart';
 import 'profile.dart';
-import 'session_info.dart';
 
 /// This class represents an account.
 class Account {
   /// The default constructor.
   Account({
     required this.emails,
-    this.created,
     this.createdTimestamp,
     this.isActive,
     this.isRegistered,
     this.isVerified,
     this.lastLoginTimestamp,
     this.lastUpdatedTimestamp,
     this.loginProvider,
-    this.oldestUpdateTimestamp,
+    this.oldestDataUpdateTimestamp,
     this.profile,
-    this.registered,
     this.registeredTimestamp,
     this.signatureTimestamp,
-    this.socialProviders,
+    this.socialProviders = const <String>[],
     this.uid,
     this.uidSignature,
-    this.verified,
     this.verifiedTimestamp,
   });
 
   /// Construct an account from the given [json].
   factory Account.fromJson(Map<String, dynamic> json) {
-    final Map<String, dynamic>? emails = json['emails'] == null ? null : json['emails'].cast<String, dynamic>() as Map<String, dynamic>;
-    final Map<String, dynamic>? profile = json['profile'] == null ? null : json['profile'].cast<String, dynamic>() as Map<String, dynamic>;
+    final Map<String, dynamic>? emails = json['emails'] as Map<String, dynamic>?;
```

Comment:
@sagishm I noticed that you had changed to parsing of the emails / profile to handle the null case and casting to a Map.

Casting to a nullable Map, using `as Map<String, dynamic>?` should already handle both the null and Map<String, dynamic> cases (the `?` handles nullable types). Also, using `cast<K, V>()` on a Map and then doing _another_ cast using `as`, is redundant. So I restored the original code here.

Reply:
Hi,
This change occurred because the code caused a parsing crash during our testing phase.
We believe it's a temporary Flutter version issue with LinkedHanshMap parsing.
We will keep it on hold until its clears and revert the change.
Thanks.

Reply:
@tal-mi I _think_ I know what might be happening. The actual map is probably a `Map<Object?, Object?>`, which is what the binary messenger of the method channel returns. That type will indeed fail the forced cast with `as`, so in that case using `cast<K,V>()` is correct. I pushed a commit that should fix it for the account class. If there are other cases, we should fix that as we go I think?

Reply:
Yeah, think that might be it. We will fix it for sure. 
We are currently going into the holidays so not many working days are available for the next two weeks.
One thing that is important. Because we are now on v1.0.0 of the plugin, breaking changes are a bit more difficult to approve due to how SAP handles customer notifications. Take that in mind for massive changes.
Of course, changes that are improving the plugin significantly will be prioritized.
Thanks again.

Reply:
I understand that any version after 1.0.0 needs to take care of breaking changes. Flutter's first party packages under `flutter/packages` follow a similar rule. The breaking changes in this PR should be relatively minor. I'll follow up with smaller PR's for the remaining web API's.

The web portion of the plugin will stay experimental until we can get all the major features from the mobile SDK implemented.

Enjoy the holidays!